build-package

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+cd $(dirname ${BASH_SOURCE[0]})
+
+# Some variables
+DEBVERSION=1.99.5-1
```

Comment:
Isn't it possible to provide something that says `2.0.0-something`?! It's rather ugly and confusing to have `1.99`, don't you think? (Also, why `-1`?)

I'd go further and say let's make this currently for 1.5.0 and as soon as 2.0.0 is official (hopefully very soon), we make the switch.